### ISSUE TITLE
889 Error

### ISSUE THREAD

**Issue by johnbeallgsa** [GSA/889-tool#165]
A user is reported that when they type 'Eastern Michigan" into the the 889 Tool that we get an error message stating, "Sorry, we weren't able to connect to SAM.gov, Please try again later. 

The rest of the tool seems to be working - screenshots below.

**Comment by johnbeallgsa** [comment]
![Image](https://github.com/GSA/889-tool/assets/116584037/2272ac8d-39d4-4314-909e-c035615f0fc7)

**Comment by johnbeallgsa** [comment]
If not results is found - it should look like this. 



![Image](https://github.com/GSA/889-tool/assets/116584037/9727bbbc-8913-4ed1-8f59-d258ed747136)

**Comment by johnbeallgsa** [comment]
If results are found, it should look like this. 



![Image](https://github.com/GSA/889-tool/assets/116584037/f83bd92d-d949-49f2-bb9c-5079ff8d16ff)

**Comment by CodyHinze** [comment]
This can also be recreated by using other words followed by a state.  All of the following will return the error screen:
"test california" or "justice california"
"blank oregon"
"Test Wisconsin" 

Some query strings likely return a specific error message, but it appears the site generically displays the error that it could not connect for all errors.

**Comment by felder101** [comment]
Looks like errors in the 889 Tool is caused by the data within sam.gov. Tested API in postman and getting the error below. There is currently a warning on the API document page regarding incorrect data (https://open.gsa.gov/api/entity-api/).

I was able to recreate the error on successful searches by navigating to different page results. For example if I search "Michigan" it return results, however if I navigate to any result page beyond page 2 I get the same error.



![Image](https://github.com/GSA/889-tool/assets/30899941/3dae5094-dfa8-4846-8d60-29d076aa8a76)

**Comment by johnbeallgsa** [comment]
Great catch. To verify, this is a SAM.gov issue? Hopefully, When they fix it, the 889 tool will return to its normal function.  Is that accurate?

**Comment by felder101** [comment]
Yes, I believe this is a SAM.gov issue due to the message they are displaying within the api documentation and the fact that some searches return successful but when accessing a different page for the same search errors are being returned. Will continue to monitor SAM.gov for updates and update when issue has been addressed. I believe when they fix the issue the application will run without errors.

**Comment by LoraBradford** [comment]
Here are two more vendors that are getting the error message. I did find that I could get to Fisher Scientific if I just search Fisher.

Fisher Scientific and Illumina

**Comment by JennaySDavis** [comment]
This issue has been resolved, closing ticket.
